lib/google_drive.rb

```diff
@@ -122,34 +113,24 @@ def self.saved_session(path = nil, proxy = nil, client_id = nil, client_secret =
           :application_version => "0.4.0"
       )
       auth = client.authorization
-      auth.client_id = client_id
-      auth.client_secret = client_secret
-      auth.scope = [
-          "https://www.googleapis.com/auth/drive",
-          "https://spreadsheets.google.com/feeds/"
-      ]
-      auth.redirect_uri = "urn:ietf:wg:oauth:2.0:oob"
-
-      if token_data
+      auth.client_id = config.client_id
+      auth.client_secret = config.client_secret
+      auth.scope = config.scope.join(' ')
```

Comment:
You can specify an array here, so you don't need to join.

Also, please use:

    ["https://www.googleapis.com/auth/drive", "https://spreadsheets.google.com/feeds/"]

as the default when this is omitted in the config file.